### PR TITLE
Open source contribution guide

### PR DESCRIPTION
**Pull request by WojciechBarczynski**

### CODE REVIEW

File: guide/contributing_guide.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+- after passing the review, the PR will be merged
+<br>
+
+If you wish to create a new plugin you can give us a shout and if it's something we want as a part of our ecosystem we'll create a repo for you, guide you on your work and maintain the plugin in future. 
+<br>
+However, if feel confident enough to maintain it on your own, you can, of course, create your own repo and hex package - we only ask you to follow the naming conventions of the framework package and modules. In that case, don't forget to let us know about your work so we could include it on the list of available plugins.
+
+## Code style guide
 
 - Use Elixir formatter (`mix format`). `.formatter.exs` should contain `:membrane_core` in `:import_deps`.
```
Comment (DominikWolek) [inline guide/contributing_guide.md:22]: Maybe this code style guide should start with basing the package on [membrane_template_plugin](https://github.com/membraneframework/membrane_template_plugin)? I'm not sure if contributors should modify formatter config from the template.

File: guide/contributing_guide.md [hunk 2]
```diff
@@ -1,4 +1,23 @@
...
+- fork it
+- apply change on some branch with a meaningful name
+- create a PR from fork to our repo
+- await feedback from someone from the team
+- after passing the review, the PR will be merged
+<br>
+
+If you wish to create a new plugin you can give us a shout and if it's something we want as a part of our ecosystem we'll create a repo for you, guide you on your work and maintain the plugin in future. 
+<br>
+However, if feel confident enough to maintain it on your own, you can, of course, create your own repo and hex package - we only ask you to follow the naming conventions of the framework package and modules. In that case, don't forget to let us know about your work so we could include it on the list of available plugins.
```
Comment (DominikWolek) [inline guide/contributing_guide.md:18]: Those paragraphs feel odd to me. Maybe double `<br>` or no `<br>` at all?

File: guide/contributing_guide.md (hunk 1 above)
Comment (WojciechBarczynski) [reply guide/contributing_guide.md]: Fixed, check it out and let me know if's ok right now

File: guide/contributing_guide.md (hunk 2 above)
Comment (WojciechBarczynski) [reply guide/contributing_guide.md]: Fixed

**Review by DominikWolek — APPROVED** [review]

**Comment by DominikWolek** [comment]
Merge master and we can close this PR

**Pull request merged**